test/SlowTests/Issues/RavenDB_9519.cs

```diff
@@ -17,16 +17,25 @@ public class RavenDB_9519 : RavenTestBase
         [Fact]
         public async Task NestedObjectShouldBeExportedAndImportedProperly()
         {
+            var id = "companies/1";
+            string cv;
+
             using (var store = GetDocumentStore())
             {
                 using (var session = store.OpenSession())
                 {
-                    session.Store(_testCompany, "companies/1");
+                    session.Store(_testCompany, id);
                     session.SaveChanges();
+                    cv = session.Advanced.GetChangeVectorFor(_testCompany);
                 }
 
                 var client = new HttpClient();
                 var stream = await client.GetStreamAsync($"{store.Urls[0]}/databases/{store.Database}/streams/queries?query=From%20companies&format=csv");
+                
+                using (var session = store.OpenSession())
+                {
+                    session.Delete(id);
```

Comment:
Why to delete it? You added the assertion that CV has changed so we'll be sure the doc was overwritten

Reply:
I guess that this code could be removed, because SaveChanges is not called anyway

Reply:
Right, I didn't notice that :)